seedrandomizer feature

gothitachisama opened the pull request:
added seed randomizer button
added auto randomizer checkbox

Closes #39

gothitachisama committed:
seedrandomizer feature

added seed randomizer button
added auto randomizer checkbox

tocram1 commented:
This looks good to me

nousr commented:
Hey, to keep things simple (and consistent with colloquial methods)--why don't we have Seed=-1 mean "use a random seed"? this will remove the need for a "randomize seed" button.

I think we can keep the dedicated "randomize seed after dream" one, though--I think that's a reasonable enough solution for now.

Otherwise lgtm 🚀

gothitachisama committed:
modifications

removed button
moved button funtionality to "seed=-1"
added tooltip for seed spinbox

gothitachisama commented:
the code updated accordingly
